app/src/main/java/com/door43/translationstudio/newui/home/ImportDialog.java

```diff
@@ -232,6 +235,17 @@ private void importUri(Uri uri) {
         }
     }
 
+    private void applySourcesToProjects(Translator translator, String[] targetTranslationSlugs) {
```

Comment:
I would not handle the selected source translations in the import dialog because there are several ways a translation can get into the app. They could uninstall+reinstall the app, restore a backup, or import from a friend.

We should instead make the selection in `com.door43.translationstudio.newui.translate.TargetTranslationActivity.onCreate()`. Then we only have to add code once.